src/Control/Monad/Bayes/PopulationT.hs

```diff
@@ -65,44 +65,44 @@ import Numeric.Log qualified as Log
 import Prelude hiding (all, sum)
 
 -- | A collection of weighted samples, or particles.
-newtype Population m a = Population (Weighted (ListT m) a)
+newtype PopulationT m a = PopulationT (Weighted (ListT m) a)
   deriving newtype (Functor, Applicative, Monad, MonadIO, MonadSample, MonadCond, MonadInfer)
 
-instance MonadTrans Population where
-  lift = Population . lift . lift
+instance MonadTrans PopulationT where
+  lift = PopulationT . lift . lift
 
 -- | Explicit representation of the weighted sample with weights in the log
 -- domain.
-population, runPopulation :: Population m a -> m [(a, Log Double)]
-population (Population m) = runListT $ weighted m
+population, runPopulationT :: PopulationT m a -> m [(a, Log Double)]
+population (PopulationT m) = runListT $ weighted m
```

Comment:
It is more customary to have a name like `population` to be a constructor for the type. See e.g. https://hackage.haskell.org/package/transformers-0.6.0.4/docs/Control-Monad-Trans-Reader.html#v:reader. From that perspective I'd expect this type signature:

```haskell
population :: [a, Log Double] -> PopulationT m a
```

Reply:
That's true. I think this can be deferred to a separate PR, but I agree. We'd then want `runSamplerIO`, `runWeighted` and so on.